contents/errorsLabels.yml

```diff
@@ -0,0 +1,2 @@
+errors:
+  errorMatch: "Il valore inserito non è valido"
```

Comment:
I'm not particularly convinced about this file, I'd move this content in specific page file